src/RecoveryServices/RecoveryServices.Backup.Autorest/custom/Cmdlets/Policy/Edit-AzRecoveryServicesBackupRetentionPolicyClientObject.ps1

```diff
@@ -0,0 +1,1044 @@
+
+
+function Edit-AzRecoveryServicesBackupRetentionPolicyClientObject {
```

Comment:
Per my understanding, the synopsis and description of this cmdlet will miss if no synopsis and description are added for this function. Have you edited these parts in docs/Edit-AzRecoveryServicesBackupRetentionPolicyClientObject.md manually?

Reply:
no not manually. I think it's coming from line no 6 in this file ?

[Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Description('Edits the retention settings for the policy client object')]